pkgs/tools/games/jja/default.nix

```diff
@@ -0,0 +1,32 @@
+{ lib, stdenv
```

Comment:
put stdenv in the next line like the others

Reply:
done.